src/app/Startup.cs

```diff
@@ -94,7 +95,7 @@ public static void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             // map the controllers
             app.UseEndpoints(ep => { ep.MapControllers(); });
-
+            
```

Comment:
linter error

Reply:
@bartr which linter and what is the error? It looks like one space between 97 and the next block of code on 99.